src/unix/linux-syscalls.c

```diff
@@ -358,7 +358,7 @@ int uv__statx(int dirfd,
               int flags,
               unsigned int mask,
               struct uv__statx* statxbuf) {
-#if defined(__NR_statx)
+#if defined(__NR_statx) && !defined(__ANDROID__)
```

Comment:
Can you also add a comment explaining why it is disabled on Android? LGTM apart from that.